### ISSUE TITLE
Feature Request: Add Video Games as a content type and import from avsforum to catalogue

### ISSUE THREAD

**Issue by ALEX8642** [3ll3d00d/beqcatalogue#17]
Profiles posted here: https://www.avsforum.com/threads/bass-eq-for-filtered-games.3047456/

**Comment by ALEX8642** [comment]
Personal note: I'm an amateur when it comes to Python (taking a course at UofT presently to enhance skills),  but if I can help make this happen, I will.

**Comment by 3ll3d00d** [comment]
Someone has to take on the job of putting them into a repo, if this happens then they can be added

**Comment by ALEX8642** [comment]
I'll take a stab at it... From the looks of it, I would have to create another fork of beqcatalogue, and basically create the same structure used by the other contributors, and fill that with those csv/xml/md files/etc. files for video games, push the images to the BEQImages one... etc? Also follow that instruction in the main readme to push the changes live, though I imagine not until the content type for video games is in place? How close am I?

**Comment by 3ll3d00d** [comment]
no, you need to look at the source repos for how to do it, nothing to do with forking beqc

**Comment by ALEX8642** [comment]
I appreciate your patience. Can you provide a link to one of these ``source repos`` as a starting reference point?

**Comment by 3ll3d00d** [comment]
you'll see them listed in https://github.com/3ll3d00d/beqcatalogue/blob/master/.github/workflows/update.yaml#L50
